app/views/partials/foyer/individu-form.html

```diff
@@ -184,21 +184,24 @@
     </div>
   </div>
 
-  <div class="form-group from-group-lg" ng-if="captureTauxInvalidite()">
+  <div class="form-group from-group-lg" ng-if="capturetauxIncapacite()">
     <label class="control-label col-sm-4" for="eligibiliteAss">
-      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'invalidité ?
+      Quel est {{ isDemandeur() ? 'votre' : 'son' }} taux d'incapacité ?
+      <div class="small">
```

Comment:
Ok, removing the help text for "conditions particulières".
I'm keeping the *taux* help text on the left, as I can't quickly get a nice result on the right side.